tests/networkpolicy_test.go

```diff
@@ -1,173 +1,264 @@
 package tests_test
 
 import (
+	"fmt"
 	"time"
 
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
-	v1network "k8s.io/api/networking/v1"
-	"k8s.io/apimachinery/pkg/api/errors"
-	v13 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	"github.com/pkg/errors"
+
+	corev1 "k8s.io/api/core/v1"
+	networkv1 "k8s.io/api/networking/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
```

Comment:
In the tests (and maybe in kubevirt in general) I see this is the "default" `errors` and not the one on line 10.
This may be confusing, so I suggest you switch them.

Reply:
Done

Reply:
Closing new "%w" golang 1.13 expression is used to wrap error.